dir-spec.txt

```diff
@@ -2125,6 +2125,17 @@
         to read.
         First-appeared: Tor 0.3.5.1-alpha.
 
+    "bandwidth-file" SP digest NL
+
+        [At most once for votes; does not occur in consensuses.]
+
+        A digest of the bandwidth file used to generate this vote.
+        (Hex, upper-case.)
+
+        This line appears in this vote in the the same cases the
```

Comment:
Should we still include the digests if the bandwidth file is not parseable?
(Should CollecTor still archive the file?)

If we don't, it will be harder to debug bandwidth scanners.
If we do, and the file contains sensitive content, then CollecTor archiving it is a security risk.

I think we should publish the file and its digest, because debugging is a much more common issue.

```suggestion
        If an authority is not configured with a V3BandwidthsFile, this line
        SHOULD NOT appear in its vote.

        If an authority is configured with a V3BandwidthsFile, but parsing
        fails, this line SHOULD appear in its vote, with the digest(s) of the
        unparseable file.
```